sklearn/feature_extraction/tests/test_feature_hasher.py

```diff
@@ -43,13 +43,15 @@ def test_feature_hasher_strings():
 
 
 def test_feature_hasher_pairs():
-    raw_X = (iter(d.items()) for d in [{"foo": 1, "bar": 2},
+    raw_X = (iter(d.items()) for d in [{"foo": 1, "bar": "a"},
                                        {"baz": 3, "quux": 4, "foo": -1}])
     h = FeatureHasher(n_features=16, input_type="pair")
     x1, x2 = h.transform(raw_X).toarray()
     x1_nz = sorted(np.abs(x1[x1 != 0]))
     x2_nz = sorted(np.abs(x2[x2 != 0]))
-    assert_equal([1, 2], x1_nz)
+    print x1_nz
+    print x2_nz
```

Comment:
I am truly sorry about that, i tough i removed that. I simply messed up with git. I'll make the update in a few hours.